src/app/log-in/log-in.component.html

```diff
@@ -50,7 +84,8 @@
       <span class="font-weight-bold">Sign up / log in with Google</span>
     </button>
     <button
-      class="button button-large button-secondary mb-40px"
+      class="mb-40px"
+      class="button button-large button-secondary mb-20px"
```

Comment:
```suggestion
      class="button button-large button-secondary mb-20px"
```
we should only have one class definition per element